EventFilter/L1TRawToDigi/plugins/implementations_stage2/EMTFUnpackerTools.cc

```diff
@@ -31,7 +31,7 @@ namespace l1t {
 							_hit.Subsector(), _hit.Ring(), _hit.CSC_ID() ) );
 
         _hit.SetCSCDetId   ( _hit.CreateCSCDetId() );
-        _hit.SetCSCLCTDigi ( _hit.CreateCSCCorrelatedLCTDigi() );
+        //_hit.SetCSCLCTDigi ( _hit.CreateCSCCorrelatedLCTDigi() );
```

Comment:
Why is this commented-out?

Reply:
OK, that makes sense.  Indeed, functions to create them on the fly is a much better solution.